test/chains/networkRequestChain.test.js

```diff
@@ -266,15 +267,17 @@ describe('Network request chain', () => {
 				},
 			},
 		});
+	});
 
-		assert.throws(() => toJSON({}), 'Throw if no comparator and strategy defined');
-		assert.throws(() => toJSON({wasMade: true}), 'Throw if no comparator defined');
-		assert.throws(() => toJSON({
+	it('should throw error in case of invalid input', async() => {
+		await testInputError(toJSON, [{}]);
```

Comment:
I would prefer avoid async/await noise. Could you implement sync version of testInputError and use async version only there where it really needed?